example/dice/rolldice.go

```diff
@@ -4,7 +4,10 @@
 package main
 
 import (
-	"go.opentelemetry.io/contrib/bridges/otelslog"
+	// TODO: "go.opentelemetry.io/contrib/bridges/otelslog".
```

Comment:
I'd rather not leave these TODOs in the example. Maybe use a tracking issue instead?

Reply:
https://github.com/open-telemetry/opentelemetry-go/issues/5801

Reply:
Also https://github.com/open-telemetry/opentelemetry-go/pull/5791/commits/ae37bbad440ff8403c24fd6ff112801fe4450ba8